app.js

```diff
@@ -31,9 +33,32 @@ app.use(expressValidator());
 /** Index page */
 // TODO: Add query to fetch most recent upcoming events
 app.get('/', (req, res) => {
-    res.render('index', {
-        events: null
-    });
+    let tweets = [];
+    const url = 'https://api.twitter.com/1.1/statuses/user_timeline.json?screen_name=enucs&exclude_replies=true&include_rts=false&count=3';
+    const bearerToken = 'bearer ' + token.bearerToken();
+    const instance = axios({url: url, headers: { 'Authorization': bearerToken }});
+    instance
+        .then((res) => {
+            res.data.forEach(tweet => {
+                let retrievedTweet = { 
+                    body: tweet.text, 
+                    created_at: new Date(tweet.created_at).toLocaleString('en-GB', { 
+                        day: '2-digit', month: 'short', year: 'numeric', hour: '2-digit', minute: '2-digit'}), 
+                    handle: tweet.user.screen_name 
+                }
+                tweets.push(retrievedTweet);
+            });
+        })
+        .then(() => {
+            console.log(tweets);
+            res.render('index', {
+                events: null,
+                tweets: tweets
+            });
+        })
+        .catch((err) => {
+            console.log(err);
```

Comment:
Maybe implement logging to a file at some point? I don't mind doing that. Would make it easier to debug if anything ever fails in a way we can't predict.

Reply:
Will create an issue for logging in a future update